caluma/Dockerfile

```diff
@@ -5,3 +5,5 @@ ENV PATH="${HOME}/.local/bin:${PATH}"
 COPY extensions /app/caluma/extensions
 COPY ci/test.sh /app/caluma/ci/test.sh
 COPY data /app/caluma/data
+COPY requirements.txt /tmp
```

Comment:
I'd copy this to /app/sagw-requirements.txt to be more explicit

Reply:
Fixed